{
    "CVE_data_meta": {
        "ASSIGNER": "security@debian.org",
        "ID": "CVE-2016-7513",
        "STATE": "PUBLIC",
        "UPDATED": "2020-06-15"
    },
    "cna-container": {
        "affected": {
            "vendors": [
                {
                    "products": [
                        {
                            "versions": [
                                {
                                    "version_value": "n/a"
                                }
                            ]
                        }
                    ],
                    "vendor_name": "n/a"
                }
            ]
        },
        "descriptions": [
            {
                "lang": "eng",
                "value": "Off-by-one error in magick/cache.c in ImageMagick allows remote attackers to cause a denial of service (segmentation fault) via unspecified vectors."
            }
        ],
        "problemtypes": {
            "descriptions": [
                [
                    {
                        "description": "n/a",
                        "lang": "eng",
                        "type": "text"
                    }
                ]
            ]
        },
        "references": [
            {
                "name": "93121",
                "refsource": "BID",
                "url": "http://www.securityfocus.com/bid/93121"
            },
            {
                "name": "[oss-security] 20160922 Re: CVE Requests: Various ImageMagick issues (as reported in the Debian BTS)",
                "refsource": "MLIST",
                "url": "http://www.openwall.com/lists/oss-security/2016/09/22/2"
            },
            {
                "name": "https://github.com/ImageMagick/ImageMagick/commit/a54fe0e8600eaf3dc6fe717d3c0398001507f723",
                "refsource": "CONFIRM",
                "url": "https://github.com/ImageMagick/ImageMagick/commit/a54fe0e8600eaf3dc6fe717d3c0398001507f723"
            },
            {
                "name": "https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=832455",
                "refsource": "CONFIRM",
                "url": "https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=832455"
            },
            {
                "name": "https://bugzilla.redhat.com/show_bug.cgi?id=1378733",
                "refsource": "CONFIRM",
                "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1378733"
            }
        ]
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "5.0"
}